llvm/lib/Target/NVPTX/NVPTXTargetMachine.cpp

```diff
@@ -59,6 +59,12 @@ static cl::opt<bool> DisableRequireStructuredCFG(
              "unexpected regressions happen."),
     cl::init(false), cl::Hidden);
 
+static cl::opt<bool> UseShortPointersOpt(
```

Comment:
This definitely should not be configurable by cl::opt

Reply:
I agree, but is that a blocking concern? I assume this is load bearing. It has tons of tests. As covered in the commit message, I modeled this as an ABI name string (which wasn't used for anything before) and moved on.